kubernetes/debugging/index.json

```diff
@@ -15,12 +15,11 @@
       "text": "finish.md"
     },
     "assets": {
-      "host01": [{"file": "*.yml", "target": "/manifests"}],
-      "host02": [{"file": "*.yml", "target": "/manifests"}]
```

Comment:
both host02 + host01 gone?

Reply:
just host2. host1 is below but I removed the comma at the end so Github just removes the line and adds a new one.